newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -210,11 +219,10 @@ export const GameDetailsDialog = ({
   );
 
   const authorUsernames =
-    publicGame && publicGame.authors
-      ? publicGame.authors
-          .map(author => (author ? author.username : null))
-          .filter(Boolean)
-      : [];
+    publicGame && publicGame.authors.map(author => author.username);
```

Comment:
some users don't have a username defined, so I would keep the `.filter(Boolean)` to ensure we don't have a `null` object in the authors (or owners) array